recipes/bzip2/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set version = "1.0.6" %}
+
+package:
+  name: bzip2
+  version: {{ version }}
+
+source:
+  fn: bzip2-{{ version }}.tar.gz
+  url: http://www.bzip.org/{{ version }}/bzip2-{{ version }}.tar.gz
+  md5: 00b516f4704d4a7cb50a1d97e6e8e15b
+  patches:
+    - Makefile.patch # Add -fpic and -fPIC to CFLAGS
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - toolchain
+
+test:
+  commands:
+    # Help
+    - bzip2 --help
+    
+    # Executables
+    {% set bzip2_executables = [
+      "bunzip2",
+      "bzcat",
+      "bzcmp",
+      "bzdiff",
+      "bzegrep",
+      "bzfgrep",
+      "bzgrep",
+      "bzip2recover",
+      "bzip2",
+      "bzless",
+      "bzmore"
+    ] %}
+    {% for executable in bzip2_executables %}
+    - test -f ${PREFIX}/bin/{{ executable }}  # [unix]
+    {% endfor %}
+    
+    # Headers
+    - test -f ${PREFIX}/include/bzlib.h  # [unix]
+    - if not exist %LIBRARY_INC%\\bzlib.h exit 1 # [win]
+    
+    # Libraries
+    - test -f ${PREFIX}/lib/libbz2.a  # [unix]
+    - test -f ${PREFIX}/lib/libbz2.so.1.0  # [unix]    
+    - test -f ${PREFIX}/lib/libbz2.so.1.0.6  # [unix]
```

Comment:
So on Mac these are `dylib` instead of `so`. Also Mac tends to put the versions before the extension instead of after. Would break these into two tests.